chat-plugins/helptickets.js

```diff
@@ -364,16 +402,23 @@ exports.pages = {
 					buf += `<p><Button>other</Button></p>`;
 					break;
 				case 'ticket':
+					if (!isLast) break;
 					buf += `<p>If you feel that staff did not properly help you with your last issue, click the button below to get in touch with an upper staff member.</p>`;
-					buf += `<p><button class="button" name="send" value="/helpticket submit Report Last Ticket">Report last ticket</button></p>`;
+					buf += `<p><Button>confirmticket</Button></p>`;
 					break;
 				case 'password':
 					buf += `<p>If you lost your password, click the button below to make a post in Admin Requests. We will need to clarify a few pieces of information before resetting the account. Please note that password resets are low priority and may take a while; we recommend using a new account while waiting.</p>`;
 					buf += `<p><a class="button" href="http://www.smogon.com/forums/forums/other-admin-requests.346/">Request a password reset</a></p>`;
 					break;
 				case 'other':
+					if (!isLast) break;
 					buf += `<p>If your issue is not handled above, click the button below to ask for a global. Please be ready to explain the situation.</p>`;
-					buf += `<p><button class="button" name="send" value="/helpticket submit Other">Call a global staff member</button></p>`;
+					buf += `<p><Button>confirmother</Button></p>`;
+					break;
+				default:
+					if (!page.startsWith('confirm')) break;
+					buf += `<p><b>Are you sure you want to submit a report? Type: ${ticketTitles[page.slice(7)]}</b></p>`;
```

Comment:
This is vague, "Type:" makes me think "You should type the following text with your keyboard:"

Maybe `submit a "${ticketTitles[page.slice(7)]}" report?`